src/SearchParameters/RefinementList.js

```diff
@@ -104,7 +103,7 @@ var lib = {
     } else if (isFunction(attribute)) {
       var hasChanged = false;
 
-      var newRefinementList = reduce(refinementList, function(memo, values, key) {
+      var newRefinementList = refinementList.reduce(function(memo, values, key) {
```

Comment:
This might be an Object 🤔

Reply:
Object here, look the reducer function 👍